frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx

```diff
@@ -234,7 +234,7 @@ export function PropertyValue({
                 makeLabel={(_, startOfRange) => (
                     <span className="hide-when-small">
                         Matches all values {operator === PropertyOperator.IsDateBefore ? 'before' : 'after'}{' '}
-                        {startOfRange}
+                        {startOfRange} if evaluated today.
```

Comment:
Today or at the time of querying?

Reply:
startOfRange parses -7d to the actual date instead of 'last 7 days' so assumes time of querying is today